langchain4j-oracle/src/test/java/dev/langchain4j/store/embedding/oracle/CommonTestOperations.java

```diff
@@ -78,21 +80,37 @@ private CommonTestOperations() {}
                         .withPassword("testpwd");
                 oracleContainer.start();
 
-                DATA_SOURCE.setURL(oracleContainer.getJdbcUrl());
-                DATA_SOURCE.setUser(oracleContainer.getUsername());
-                DATA_SOURCE.setPassword(oracleContainer.getPassword());
-            }
-            else {
-                DATA_SOURCE.setURL(urlFromEnv);
-                DATA_SOURCE.setUser(System.getenv("ORACLE_JDBC_USER"));
-                DATA_SOURCE.setPassword(System.getenv("ORACLE_JDBC_PASSWORD"));
+                initDataSource(DATA_SOURCE,
+                    oracleContainer.getJdbcUrl(), oracleContainer.getUsername(), oracleContainer.getPassword());
+                initDataSource(VECTOR_INDEX_DATA_SOURCE,
```

Comment:
We could call it "SYSDBA_DATA_SOURCE" to reflect the difference from the normal DATA_SOURCE. We could also consider if we need to be pooling sysdba connections; If not then we could just have a CommonTestOperations.getSysDbaConnection() method.